jekyll/_cci2/server/installation/phase-2-core-services.adoc

```diff
@@ -500,6 +500,81 @@ workloadIdentity: <workload-identity-service-account-email>
 // Stop hiding from AWS PDF
 endif::env-aws[]
 
+
+== Example Manifests
+The following are example manifests that contain the basic required parameters necessary to spin up the circleci-server helm installation.
+
+=== AWS
+
+```yaml
+
+```
```

Comment:
I'll file my AWS PR once this is merged in 👀

Reply:
👍